### DISCUSSION TITLE
I keep getting an error when trying to DL;

### DISCUSSION THREAD

**Discussion by LJD1800** [aclap-dev/video-downloadhelper#293] [Help and Q&A]
### Your question

Everything was fine until today. Could you please help with this error I am getting anytime I try to DL a video? The video I was trying to DL is not public & requires a log in.  I was hoping the error itself might have a clue.

ENOENT: no such file or directory, mkdir '\\?'
Error: ENOENT: no such file or directory, mkdir '\\?'
    at rs.receive (chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:1:13157)
    at chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:1:42625

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.1.1 (stable)
Browser locale: en-US
Production build
Built on 2024-02-19
Build options: browser=edge, linuxlic=false, noyt=false, target=microsoft

Platform Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0.0

Found companion app: VdhCoApp 2.0.19
Companion app binary: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by Mattmjs** [comment]
Perhaps a folder / directory issue, did you try a different location ? I'm guessing though.
And can you share the hit details information which includes master and media manifest.

**Reply by LJD1800** [reply]
You were absolutely correct.  It was a directory issue which I discovered shortly after my post. Embarrassed but have to admit it was a stupid human trick & I was the human.  Thanks for your help though.  I really appreciate you taking the time.

**Comment by Mattmjs** [comment]
@LJD1800 , my initial response was about a week ago. Did you get notified of this response ?
Has the problem been resolved or does it persist ?
